freqtrade/optimize/optimize_reports/optimize_reports.py

```diff
@@ -25,8 +25,8 @@
 
 
 def generate_trade_signal_candles(
-    preprocessed_df: Dict[str, DataFrame], bt_results: Dict[str, Any]
-) -> DataFrame:
+    preprocessed_df: Dict[str, DataFrame], bt_results: Dict[str, Any], analysis_on="open_date"
```

Comment:
i'd omit the "open_date" default.
all usages i can see ad-hock seem to provide it - and i'd not consider this function as external interface (which people import in their custom code).

Reply:
makes sense.. also do you feel `analysis_on` is the correct variable name here? have been struggling with terminologies in this PR. :)

Reply:
i think in other contexts, we've used "column" or "date_col" (after quickly looking through the metrics.py file ...

Reply:
cool.. i think i ll go with `date_col`. Thanks.